pyproject.toml

```diff
@@ -38,6 +38,8 @@ classifiers = [
 dependencies = [
     "numpy",
     "typer",
+    "einops",
+    "napari",
```

Comment:
napari is a big dependency and only necessary for vis, consider removing (or gating behind a [vis] selector, can provide details)